modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/classvars.mustache

```diff
@@ -100,6 +100,8 @@
 
     additional_properties_type = {{#additionalPropertiesType}}({{{additionalPropertiesType}}},)  # noqa: E501{{/additionalPropertiesType}}{{^additionalPropertiesType}}None{{/additionalPropertiesType}}
 
+    _nullable = {{#isNullable}}True{{/isNullable}}{{^isNullable}}False{{/isNullable}}
```

Comment:
Do we have a test where we test this new feature with `assert None == NullableClass(None)`?
Where?
If we don't have it, can you add it?

Reply:
I'm going to cover the composed model in this PR. BTW, this is only relevant to the oneOf/anyOf children, for other cases it's already covered in the existing code and unit tests.

Reply:
See 8fdbd278fdbdf1da674206cf1ca8d337a7c26d80